2020-04-02 23:17:46,252 ERROR: Exception on /articles [GET] [in /home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./edmondsonline/landing/routes.py", line 32, in articles
    title=_('Articles'))
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/khammrich/test-edmondsonline/edmondsonline/templates/landing/articles.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/khammrich/test-edmondsonline/edmondsonline/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/khammrich/test-edmondsonline/edmondsonline/templates/base.html", line 55, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/khammrich/test-edmondsonline/edmondsonline/templates/landing/articles.html", line 9, in block "app_content"
    <h1><li><a href="{{ url_for('landing.so_we_are_here') }}">{{ _('Explore') }}</a></li></h1>
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/helpers.py", line 356, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/app.py", line 2061, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/flask/helpers.py", line 345, in url_for
    force_external=external)
  File "/home/khammrich/test-edmondsonline/venv-edtest/lib/python3.6/site-packages/werkzeug/routing.py", line 1776, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'landing.so_we_are_here'. Did you mean 'landing.art1' instead?
2020-04-02 23:18:14,089 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:19:14,485 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:23:06,775 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:24:27,648 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:26:39,997 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:27:23,159 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:27:36,801 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:29:44,159 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:30:06,646 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:30:46,773 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:31:14,882 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:33:18,890 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:34:53,294 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:37:33,753 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:38:27,597 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:38:58,750 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:39:44,192 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:40:14,841 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:40:25,706 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:40:43,181 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:41:30,784 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:42:25,352 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:43:00,562 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:43:52,269 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:44:38,176 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-02 23:46:21,100 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:05:42,871 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:18:06,109 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:18:41,971 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:20:08,514 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:20:16,079 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:20:39,240 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:21:07,714 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:21:55,643 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:23:58,996 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
2020-04-03 00:35:43,406 INFO: Microblog startup [in ./edmondsonline/__init__.py:79]
